python/dlr/tf_model.py

```diff
@@ -0,0 +1,153 @@
+# coding: utf-8
+import os
+import tensorflow as tf
+
+# A prefix that will be prepended to the names in graph_def
+PREFIX = "import"
+
+
+class TFModelImpl:
+    """
+    TFModelImpl is a wrapper on top of tensorflow which implements DLRModel API
+
+    Parameters
+    ----------
+    frozen_graph_file : str
```

Comment:
frozen_graph_file -> model_path/model_dir? since probably we will support savedmodel or other types of format with this parameter

Reply:
DLC compiler might support input different formats, e.g. saved model and frozen graph. After the "compilation" DLC will produce TF frozen graph as an output. So, `TFModelImpl` only needs to support frozen graph format.